code/modules/jobs/access.dm

```diff
@@ -13,21 +13,21 @@
 	if(isAdminGhostAI(accessor))
 		//Access can't stop the abuse
 		return TRUE
-	else if(istype(accessor) && SEND_SIGNAL(accessor, COMSIG_MOB_ALLOWED, src))
+	//If the mob has the simple_access component with the requried access, we let them in.
+	else if(SEND_SIGNAL(accessor, COMSIG_MOB_TRIED_ACCESS, src) & ACCESS_ALLOWED)
```

Comment:
why are you creating a new signal when COMSIG_OBJ_ALLOWED already exists and seems to have the same purpose?

Reply:
I've renamed COMSIG_MOB_ALLOWED.